### PR TITLE
Feature/obstacle detector

### PR DESCRIPTION
**Pull request by xleelz**
A lot of code is commented out because it is not finished, but the node currently takes it's input, filters it, and publishes the filtered cloud to the obstacles topic.

### CODE REVIEW

File: cr17/CMakeLists.txt~ [hunk 1]
```diff
@@ -0,0 +1,212 @@
+cmake_minimum_required(VERSION 2.8.3)
```
Comment (JacobAMason) [inline cr17/CMakeLists.txt~]: I don't believe this file should be included. This might be a cached file.

File: cr17/package.xml~ [hunk 2]
```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0"?>
```
Comment (JacobAMason) [inline cr17/package.xml~]: Don't think this file belongs, either.

**Review by JacobAMason — CHANGES_REQUESTED** [review]
Remove the files ending with a tilde (~), because I'm pretty sure they are cached files.

File: cr17/CMakeLists.txt~ (hunk 1 above)
Comment (bgrayd) [reply cr17/CMakeLists.txt~]: yes, it is a cached file from gedit

File: cr17/package.xml~ (hunk 2 above)
Comment (bgrayd) [reply cr17/package.xml~]: yes, it is a cached file from gedit

**Comment by JacobAMason** [comment]
Might also want to squash theses commits, since there are two commits with the same name.

**Review by JacobAMason — APPROVED** [review]

**Comment by JacobAMason** [comment]
Needs input from @ryan-smith before merging.

**Pull request closed without merging**